src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int BallLimit = 3;
```

Comment:
let's make it a constant
constants are final static fields that have a PROPER_NAMING pattern

Reply:
When I add static I have a mistake...
"Modifier 'static' not allowed here"
Should I commit it anyway? And can you explain why it is constant?...

Reply:
> let's make it a constant constants are final static fields that have a PROPER_NAMING pattern

When I add static I have a mistake...
"Modifier 'static' not allowed here"
Should I commit it anyway? And can you explain why it is constant?...

Reply:
I add static and "All checks have failed"...